content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,214 @@
+
+```yaml
+triggers:
+- type: azure-data-explorer
+  metadata:
+    podIdentity: "azure" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    clientId: "AAD_APP_CLIENT_ID" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    clientSecret: "AAD_APP_SECRET" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    tenantId: "AAD_APP_TENANT_ID" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    databaseName: "DATA_EXPLORER_DATABASE_NAME"
+    endpoint: "https://<ClusterName>.<Region>.kusto.windows.net"
+    query: |
+      StormEvents
+      | summarize StormCount = count(), TypeOfStorms = dcount(EventType) by State
+      | top 1 by StormCount desc
+    threshold: "1000"
+    metricName: "DATA_EXPLORER_METRIC_NAME" # Optional
+    # Alternatively, you can use existing environment variables to read aad app creds from:
+    clientIdFromEnv: AAD_APP_CLIENT_ID_ENV_VAR_NAME # Optional. You can use this instead of `clientId` parameter.
+    clientSecretFromEnv: AAD_APP_SECRET_ENV_VAR_NAME # Optional. You can use this instead of `clientSecret` parameter.
+    tenantIdFromEnv: AAD_APP_TENANT_ID_ENV_VAR_NAME # Optional. You can use this instead of `tenantId` parameter.
+```
+
+This scaler is backed by Azure Data Explorer Goland SDK. Please see [this](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/api/golang/kusto-golang-client-library) page
+for further details.
+
+**Parameter list:**
+Provide either podIdentity or aad app creds (clientId, clientSecret and tenantId).
+- `clientId` - Id of the application from your Azure AD Application/service principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your service principal.
+- `clientSecret` - Password from your Azure AD Application/service principal.
+- `tenantId` - Id of the Azure Active Directory tenant. Follow [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) link to 
+retrieve your tenant id. 
+- `podIdentity` - [Azure Active Directory pod-managed identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) can be used. Only "azure" is supported. 
+- `databaseName` - The name of the Data Explorer Database to query. Use the [Azure Command Line Interface](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) to run `az kusto database list` to see a list of available databases in your cluster.
+- `endpoint` - The endpoint to query your Data Explorer Cluster. Follow [this](https://docs.microsoft.com/en-us/cli/azure/kusto/cluster?view=azure-cli-latest#az-kusto-cluster-show) link to get your cluster's endpoint.
+- `metricName` - Name to assign to the metric. (Optional, if not set KEDA will generate a name based on the database name)
+- `threshold` - Value that is used as a threshold to calculate # of pods for scale target.
```

Comment:
Changing logical ordering and removing podIdentity
```suggestion
- `endpoint` - The endpoint to query your Data Explorer Cluster. Follow [this](https://docs.microsoft.com/en-us/cli/azure/kusto/cluster?view=azure-cli-latest#az-kusto-cluster-show) link to get your cluster's endpoint.
- `databaseName` - The name of the Data Explorer Database to query. Use the [Azure Command Line Interface](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) to run `az kusto database list` to see a list of available databases in your cluster.
- `threshold` - Value that is used as a threshold to calculate # of pods for scale target.
- `tenantId` - Id of the Azure Active Directory tenant. Follow [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) link to 
retrieve your tenant id.
- `clientId` - Id of the application from your Azure AD Application/service principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your service principal.
- `clientSecret` - Password from your Azure AD Application/service principal.
- `metricName` - Name to assign to the metric. (Optional, if not set KEDA will generate a name based on the database name)
```

Reply:
Can you align with https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler as well please?

Feel free to move the `Follow this link` items to a sub-item of the list and rename it to `Use this guide`